Cython/Build/IpythonMagic.py

```diff
@@ -308,6 +312,12 @@ def critical_function(data):
         html_file = os.path.join(lib_dir, module_name + '.html')
         module_path = os.path.join(lib_dir, module_name + self.so_ext)
 
+        if args.cplus:
+            cythonized_file_ext = '.cpp'
+        else:
+            cythonized_file_ext = '.c'
+        cythonized_file = os.path.join(lib_dir, module_name + cythonized_file_ext)
```

Comment:
The `extensions[0].sources` below knows which file was compiled. I'd rather not have this logic here.